src/api/v4/constants.js

```diff
@@ -0,0 +1,45 @@
+var _ = require('underscore');
```

Comment:
should this file be under the  `dataviews` folder?

Reply:
In this file we have carto.operation Enum and carto.dataview.status Enum. May be we can split it in two files